main.py

```diff
@@ -221,15 +248,38 @@ def report():
 
         pre_period = [0, pre_delta - 1]
         post_period = [pre_delta, pre_delta + post_delta]
-
-        impact = getCiObject(df, pre_period, post_period)
+        try:
+          impact = getCiObject(df, pre_period, post_period)
+        except ValueError as e:
+          return render_template(
+          'report.html',
+          custom_inc=os.environ.get('CUSTOM_INCLUDES'),
+          summary='-',
+          chart="-",
+          report="-",
+          raw_data="-",
+          warnings=f"Error: {e}",
```

Comment:
is it safe to render error messages on a "public"  url?

Reply:
It is just the error that is returned in human words, so helps us understand what the issue is. But have changed it so it logs it and doesn't expose it.